rules/dummy-rule-for-codeauditor/rule.md

```diff
@@ -5,134 +5,80 @@ uri: dummy-rule-for-codeauditor
 authors:
   - title: Tom
```

Comment:
Use the proper format for author
https://github.com/SSWConsulting/SSW.Rules.Content/wiki/How-to-Add-an-Acknowledgment
```suggestion
  - title: Tom Bui
  - url: https://www.ssw.com.au/people/tom-bui
```